server/models/handlers/availability.js

```diff
@@ -1,29 +1,24 @@
 // Gets the time availabilities for a specific worker for a specific school.
-async function getWorkerAvailability(workerId, schoolId, startTime, endTime) {
-    const sampleAvailableTimes = [
-        {
-            workerTimeslotId: 1,
-            startTime: '08:00:00', 
-            endTime: '08:30:00',
-            date: '2020-10-20'
-        },
-        {
-            workerTimeslotId: 2,
-            startTime: '08:30:00', 
-            endTime: '09:00:00',
-            date: '2020-10-20'
-        },
-        {
-            workerTimeslotId: 3,
-            startTime: '09:00:00', 
-            endTime: '09:30:00',
-            date: '2020-10-20'
-        }
-    ];
+const _ = require('lodash');
+const availabilityHandler = require('../db/availability');
```

Comment:
Hi Nancy,
The current file is the availabilityHandler. This file that is being required is actually the availabilityModel. Thus, I would recommend revising the variable name to avoid potential confusion in the future.

Reply:
Hi Amy,
Could I request a clarification?: From what I understand, you recommend that I replace line 3 with `const availabilityModel = require('../db/availability');` - is my understanding correct?

I'd appreciate if you could clarify this so that I can make the recommended changes!

Reply:
Hi Nancy,
Yes, that is correct as the db file is a model rather than a handler.

Reply:
Hi Amy,
Thank you for clarifying that!

My action: I've made the following replacements:

1. `const availabilityHandler = require('../db/availability');` is changed to  `const availabilityModel = require('../db/availability');`
2.  `return availabilityHandler.getAvailabilityDetails(workerId, schoolId, startTime, endTime);`  is changed to  `return availabilityModel.getAvailabilityDetails(workerId, schoolId, startTime, endTime);`

You can now see these changes in the following locations:
line 3 of server/models/handlers/availability.js
line 14 of server/models/handlers/availability.js

Please confirm if this takes care of your suggestion!